core/nodejs6Action/Dockerfile

```diff
@@ -7,6 +7,17 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.gz"
 
 
+# install SASL libs
+RUN apt-get install -y \
+    g++ \
+    make \
+    python \
```

Comment:
@rabbah Before, the image is 470M and after it is about 692M. I can get it down to about 623M (or perhaps a _bit_ smaller) but that requires a fairly drastic change like this: cdfd63d2a6767bc984b63247c50d3609e951abe2

The trouble is that g++, make, and python are only needed during the npm install step. While I can remove these later in the Dockerfile, there is no space savings in the entire image because the adds and deletes happen in separate RUN instructions and, therefore, separate layers. However, putting everything in a shell script and executing that in a single RUN instruction allows the deletes to effect the size of the resulting image.